playground/buffer-mesh-independent.ts

```diff
@@ -0,0 +1,167 @@
+/**
+ * @title Buffer Mesh Independent
+ * @category Mesh
+ */
+import {
+  BlinnPhongMaterial,
+  Buffer,
+  BufferBindFlag,
+  BufferMesh,
+  BufferUsage,
+  Camera,
+  Engine,
+  IndexFormat,
+  Mesh,
+  MeshRenderer,
+  PointLight,
+  Vector3,
+  VertexElement,
+  VertexElementFormat,
+  WebGLEngine,
+  Script
+} from "oasis-engine";
+
+/**
+ * Script for updating ducks position, rotation, and scale.
+ */
+class RandomColorScript extends Script {
+  /** Color data. */
+  colorData: Float32Array;
+  /** Color buffer. */
+  colorBuffer: Buffer;
+
+  private _loopCount = 0;
+
```

Comment:
Does not show the effect of color change.

Reply:
Need merge this https://github.com/oasis-engine/engine/pull/450 for engine.